jdbc/run_jdbc_tests.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+if [ "$#" -ne 2 ]; then
+    echo "Exactly two arguments should be provided"
+    echo "Arg1: Provide an available port for the coordinator instance of the temporary citus cluster to perform jdbc tests"
+    echo "Arg2: Provide path to jdbc driver for PostgreSQL, (see https://jdbc.postgresql.org/ to download it)"
+
+    exit 1
+fi
+
+coordinator_port=$1
+jdbc_driver_path=$2
+
+# create & run tmp citus cluster for jdbc test 
+citus_dev stop /tmp/jdbc_test_db --port $coordinator_port
+rm -rf /tmp/jdbc_test_db
+mkdir -p /tmp/jdbc_test_db
+citus_dev make /tmp/jdbc_test_db --port $coordinator_port
+
+# print installed citus version first
+psql -p $coordinator_port -d $USER -c "select citus_version();"
```

Comment:
All cases like `$USER` & `$coordinator_path` should be quoted, `"$USER"` or `"$coordinator_path"`

Reply:
I suggest you run [ShellCheck](https://github.com/koalaman/shellcheck) (a shell script analysis tool) and try to solve all issues